ballerina/Tests/mock_service.bal

```diff
@@ -0,0 +1,59 @@
+service / on ep0 {
+
+    // Mock implementation for /audio/speech
```

Comment:
Rather than adding comments, its better to add doc-comments for the mock-service. Lets follow this [1] structure.

[1] - https://github.com/ballerina-platform/module-ballerinax-openai.images/blob/d723b74a3b6d84224c3aeba5911b93e3c336c736/ballerina/tests/mock_service.bal